Sources/UIExtensions/UIKitExtension.swift

```diff
@@ -2,6 +2,13 @@
 import UIKit
 
 public extension UITableView {
+
+    #if swift(>=4.2)
+    typealias RowAnimation = UITableView.RowAnimation
+    #else
+    typealias RowAnimation = UITableViewRowAnimation
+    #endif
```

Comment:
I think this is the way to go, but is `typealias RowAnimation = UITableView.RowAnimation` needed in the Swift 4.2 or over?

Reply:
Nope. Added just to create symmetry.